### DISCUSSION TITLE
not loading wallet chia on my laptop

### DISCUSSION THREAD

**Discussion by eugent1987** [Chia-Network/chia-blockchain#6561] [Support]
hello i need help , i have e problem with chia wallet not loadet app on my laptop

**Comment by loppefaaret** [comment]
If you simply cannot sync the wallet, the 1.1.7 for windows users is the fix
On the other hand, you got a spinning circle, and the text "Connecting to wallet..." it is likely a corrupted wallet database
The way around this is to remove the current wallet database:
Close down the Chia app, and open up a PowerShell window, and execute the following command:
`del ~\.chia\mainnet\wallet\db\block*`
Then try and open up the Chia app again

**Comment by BrandtH22** [comment]
@eugent1987 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
